### PR TITLE
V8 v8.5 release blog post

### PR DESCRIPTION
**Pull request by zeynepCankara**
new V8 release

### CODE REVIEW

File: src/blog/v8-release-85.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
+---
+title: 'V8 release v8.5'
+author: 'Sathya Gunasekaran'
```
Comment (gsathya) [inline src/blog/v8-release-85.md:3]: Change to your name

Comment (RReverser) [reply src/blog/v8-release-85.md:3]: And add a funny/pun tagline like in other posts please.

File: src/blog/v8-release-85.md [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+avatars:
+date: 2020-07-21
+tags:
+ - release
+description: 'V8 release v8.5'
```
Comment (RReverser) [inline src/blog/v8-release-85.md:8]: This should include a short description of features, see other posts for examples.

File: src/blog/v8-release-85.md [hunk 3]
```diff
@@ -0,0 +1,126 @@
+---
+title: 'V8 release v8.5'
+author: 'Sathya Gunasekaran'
+avatars:
```
Comment (RReverser) [inline src/blog/v8-release-85.md:4]: Add your avatar to `img` folder in 2 sizes and reference here.

File: src/blog/v8-release-85.md [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+date: 2020-07-21
+tags:
+ - release
+description: 'V8 release v8.5'
+tweet:
+---
+Every six weeks, we create a new branch of V8 as part of our [release process](https://v8.dev/docs/release-process). Each version is branched from V8’s Git master immediately before a Chrome Beta milestone. Today we’re pleased to announce our newest branch, [V8 version 8.5](https://chromium.googlesource.com/v8/v8.git/+log/branch-heads/8.5), which is in beta until its release in coordination with Chrome 85 Stable in several weeks. V8 v8.5 is filled with all sorts of developer-facing goodies. This post provides a preview of some of the highlights in anticipation of the release.
+
+JavaScript
```
Comment (RReverser) [inline src/blog/v8-release-85.md:13]: These should be formatted as headings.

File: src/blog/v8-release-85.md [hunk 5]
```diff
@@ -0,0 +1,127 @@
...
+    local.get $x
+    local.get $y
+    i64.add)
+  (export "add" (func $add)))
+```
+
+From JavaScript, only BigInts can be passed as I64 parameter:
+
+```javascript
+WebAssembly.instantiateStreaming(fetch(‘i64.wasm’))
```
Comment (RReverser) [inline src/blog/v8-release-85.md:122]:
```suggestion
WebAssembly.instantiateStreaming(fetch('i64.wasm'))
```

File: src/blog/v8-release-85.md (hunk 2 above)
Comment (zeynepCankara) [reply src/blog/v8-release-85.md]: Changed it

File: src/blog/v8-release-85.md [hunk 6]
```diff
@@ -0,0 +1,137 @@
...
+
+From JavaScript, only BigInts can be passed as I64 parameter:
+
+```javascript
+WebAssembly.instantiateStreaming(fetch('i64.wasm'))
+  .then(({ module, instance })) => {
+    instance.exports.add(12n, 30n); // result = 42n;
+    instance.exports.add(12, 30); // throws TypeError, parameters are not of type BigInt.
+  });
+```
```
Comment (mathiasbynens) [inline src/blog/v8-release-85.md:137]: The V8 API section is missing (see other release notes)

File: src/blog/v8-release-85.md [hunk 7]
```diff
@@ -0,0 +1,143 @@
...
+
+Note that, unlike mathematical and bitwise compound assignment operators, logical assignment operators only conditionally perform the assignment. Please read our explainer for a more in-depth explanation.
+
+## WebAssembly
+
+### Liftoff shipped on all platforms
+
+Since v6.9, [Liftoff](https://v8.dev/blog/liftoff) has been used as the baseline compiler for WebAssembly on Intel platforms (Chrome enabled it on desktop systems since M-69). Since we were concerned about memory increase (because of more code being generated by the baseline compiler), we held it back for mobile systems so far. After some experimentation in the last months, we are confident that the memory increase is negligible for most cases, hence we finally enable Liftoff by default on all architectures, bringing increased compilation speed, especially on arm devices (32- and 64-bit). Chrome follows along and ships Liftoff in M-85.
+Multi-value support shipped
```
Comment (mathiasbynens) [inline src/blog/v8-release-85.md:81]: Should be a header. Are you testing + previewing this locally? Doing so might make it easier to spot these kinds of things

File: src/blog/v8-release-85.md (hunk 3 above)
Comment (mathiasbynens) [reply src/blog/v8-release-85.md:4]: @zeynepCankara Only the images in `src/_img/avatars/*` are needed; please remove the ones you added in `src/_img/*.jpg`. Also, could you check the dimensions of the avatar images (it doesn't look like they're square)? They should match the other avatars.

File: src/blog/v8-release-85.md [hunk 8]
```diff
@@ -0,0 +1,145 @@
+---
+title: 'V8 release v8.5'
+author: 'Zeynep Cankara, tracking some Maps'
+avatars:
+ - 'zeynep-cankara'
+date: 2020-07-21
+tags:
+ - release
+description: 'V8 release v8.5 features Promise.any, AggregateError, String.prototype.replaceAll, logical assignment operators, Liftoff shipped on all platforms, multi-value support shipped and support for JS BigInts'
```
Comment (RReverser) [inline src/blog/v8-release-85.md:9]: Let's reword to shorten this up a bit.

Comment (RReverser) [reply src/blog/v8-release-85.md:9]: Maybe at least

```suggestion
description: 'V8 release v8.5 features Promise.any, .replaceAll on strings, logical assignment operators, WebAssembly multi-value and BigInt support, and performance improvements'
```

Comment (mathiasbynens) [reply src/blog/v8-release-85.md:9]: Could also use the `String#replaceAll` notation (we're doing it all over the place on v8.dev)

File: src/blog/v8-release-85.md [hunk 9]
```diff
@@ -0,0 +1,145 @@
...
+  // All of the promises were rejected.
+  console.assert(error instanceof AggregateError);
+  // Log the rejection values:
+  console.log(error.errors);
+}
+```
+
+If all input promises are rejected, the resulting promise is rejected with an `AggregateError` object containing an `errors` property which holds an array of rejection values.
+
+Please see our explainer for more.
```
Comment (mathiasbynens) [inline src/blog/v8-release-85.md:41]: Missing link?

File: src/blog/v8-release-85.md [hunk 10]
```diff
@@ -0,0 +1,145 @@
...
+// Works, but requires escaping inside regular expressions.
+queryString.replace(/\+/g, ' ');
+// → 'q=query string parameters'
+
+// Simpler!
+queryString.replaceAll('+', ' ');
+// → 'q=query string parameters'
+```
+
+Please see our explainer for more.
```
Comment (mathiasbynens) [inline src/blog/v8-release-85.md:59]: Missing link?

File: src/blog/v8-release-85.md (hunk 8 above)
Comment (RReverser) [reply src/blog/v8-release-85.md:9]: Oh yeah right, that's probably even better.

Comment (RReverser) [reply src/blog/v8-release-85.md:9]: Updated.

Comment (RReverser) [reply src/blog/v8-release-85.md:9]: (this got accidentally reverted in a follow-up commit, marking as unresolved)

**Pull request merged**